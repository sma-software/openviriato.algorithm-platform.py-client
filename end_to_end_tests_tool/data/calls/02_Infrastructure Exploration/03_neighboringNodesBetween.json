{
    "restResource": "/nodes",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "HasSectionTrackFromNodeID",
            "value": "7",
            "optional": false
        },
        {
            "key": "HasSectionTrackToNodeID",
            "value": "10",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetNeighboringNodesBetween(fromNodeID, toNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64" ],
    "subcategory": "Get Neighboring Nodes Between"
}
