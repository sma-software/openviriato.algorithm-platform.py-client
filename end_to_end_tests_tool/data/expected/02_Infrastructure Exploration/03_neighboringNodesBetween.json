[
  {
    "id": 15,
    "code": "45ZSAA",
    "nodeTracks": [
      {
        "id": 16,
        "code": "1",
        "debugString": "stationtrack:45ZSAA_1"
      },
      {
        "id": 17,
        "code": "2",
        "debugString": "stationtrack:45ZSAA_2"
      }
    ],
    "debugString": "station:45ZSAA"
  },
  {
    "id": 24,
    "code": "45ZWAA",
    "nodeTracks": [
      {
        "id": 25,
        "code": "1",
        "debugString": "stationtrack:45ZWAA_1"
      },
      {
        "id": 26,
        "code": "2",
        "debugString": "stationtrack:45ZWAA_2"
      }
    ],
    "debugString": "station:45ZWAA"
  }
]